pinot-core/src/main/java/org/apache/pinot/core/operator/query/DictionaryBasedAggregationOperator.java

```diff
@@ -44,6 +44,8 @@
 import org.apache.pinot.spi.data.FieldSpec;
 import org.apache.pinot.spi.utils.ByteArray;
 
+import static org.apache.pinot.core.operator.query.AggregationOperatorUtils.toDouble;
```

Comment:
Suggest not using static function import for readability. Since we already put the rule in the checkstyle, let's not adding this exception

Reply:
Can we talk about this with a wider audience? IMO Pinot's styling enforces some quite ugly code in places (line wrapping and unnecessary qualification). The real problem here would be if `toDouble` did anything _but_ convert a value to a double, so it belongs in a global namespace. I am of a democratic disposition, of course.

Reply:
Sure, I'm open to other conventions as long as it improves readability. Pinot Style mostly follows the [LinkedIn Java Style Guide](https://www.linkedin.com/pulse/coding-standard-programming-best-practices-java-part-1-david-rios/) (wildcard and static import is not permitted) which is mostly extended from [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html). The reason for certain rules that seems too strict and cause ugly code is to prevent developers from misusing them and causing poor readability. We can open a separate issue to discuss it.

Reply:
I've used Google Java format a lot in the past, and it permits static imports. Like a lot of language features (questionable uses of anonymous classes abound in the code base) they can be abused, but they're in the language for a reason. A blanket ban is better than debating the merits of every single static import, of course ;)